src/schema/ghga.yaml

```diff
@@ -693,3 +715,22 @@ slots:
   value type:
     description:
       The semantic type for the value of an attribute.
+
+enums:
+  gender_enum:
+    description: gender
+    permissible_values:
+      "XX":
```

Comment:
Why `XX` instead of `male`? Is this to emphasize on chromosomal sex instead of biological sex?

Reply:
Exactly. As this is also an ongoing discussion in the scientific community I would also opt for two properties: "gender" and "sex". gender defining the donors perception, and sex defining the biological sex. Let me know what you think

Reply:
Perfect! Yes, let's differentiate between gender and sex. 

It could well be that when a data submitter submits metadata they may have information about either gender or sex or both or none.